packages/oas-resolver/index.js

```diff
@@ -10,9 +10,11 @@ const yaml = require('yaml');
 const jptr = require('reftools/lib/jptr.js').jptr;
 const recurse = require('reftools/lib/recurse.js').recurse;
 const clone = require('reftools/lib/clone.js').clone;
+const shallowClone = require('reftools/lib/clone.js').shallowClone;
 const deRef = require('reftools/lib/dereference.js').dereference;
 const isRef = require('reftools/lib/isref.js').isRef;
 const common = require('oas-kit-common');
+const OUTER_PROPS_SUPPORTED_VERSIONS = ['3.1'];
```

Comment:
can we provide this list also from the options?

Reply:
If we can provide this array from the options and read the options to find out which versions support outer props it will be easier to expand this list in the future. Otherwise, now we will have to release a new version just to add a new version to this list. @erfemega